setup.cfg

```
@@ -33,9 +33,11 @@ datalad.extensions =
 
 [options.extras_require]
 tests =
-    coverage
-    pytest
-    pytest-cov
+    coverage>=7.0
+    pytest>=7.0
+    pytest-cov>=4.0
+    tox>=3.0
```

What about doing smth like this so there is a dedicated `dev`:

```suggestion
[dev]
    %(tests)s
    tox>=3.0
```